include/benchmark/benchmark_api.h

```diff
@@ -858,6 +861,10 @@ class Fixture : public internal::Benchmark {
 #define BENCHMARK_MAIN()                   \
   int main(int argc, char** argv) {        \
     ::benchmark::Initialize(&argc, argv);  \
+    if (argc > 1) {                        \
```

Comment:
I don't really understand why this was made into a separate function in the first place as i think it was much clearer inline (albeit in a macro).

i have no massive concerns with the current approach, though i think the "returning a boolean" is cleaner... but then again what if someone doesn't want to printf but wants to do something else with the extra args?

chuck it in. we'll fix it in post.